### PR TITLE
drinkSafe instead of drink

### PR DESCRIPTION
**Pull request by Rinn**
Just replacing two calls to mafia drink with drinkSafe

### CODE REVIEW

**Review by horrible-little-slime — APPROVED** [review]

**Review by horrible-little-slime — CHANGES_REQUESTED** [review]
I changed my mind, and no longer approve of this; drinkSafe calls acquire, which then checks for a price cap. You need to either edit acquire to not blow up for pilsners, or continue not using drinkSafe for the times we drink pilsners

**Comment by Rinn** [comment]
oh i see what you mean, I'll fix that

File: src/diet.ts [hunk 1]
```diff
@@ -186,16 +179,41 @@ function fillStomach() {
...
+  try {
+    if (
+      !get("_mimeArmyShotglassUsed") &&
+      itemAmount($item`mime army shotglass`) > 0 &&
+      availableAmount($item`astral pilsner`) > 0
+    ) {
+      drinkSafe(1, $item`astral pilsner`);
+    }
+    if (
+      globalOptions.ascending &&
```
Comment (horrible-little-slime) [inline src/diet.ts:195]: :)

**Review by horrible-little-slime — APPROVED** [review]
I'd like to see `fillLiverAstralPilsner();` used where we start filling up with turkey, regardless of whether we're ascending. That may generally be the best place for it.

**Pull request merged**